webhooks/core/annotation_validation_webhook.go

```diff
@@ -46,9 +46,9 @@ func (a *AnnotationValidator) Handle(_ context.Context, req admission.Request) a
 		"request", req)
 
 	switch req.Operation {
-	case v1beta1.Create:
+	case admissionv1.Create:
```

Comment:
nice! along with the code we also need to change in the manifest files from v1beta1 to v1.